core/orchestrator.go

```diff
@@ -307,10 +307,10 @@ func (orch *orchestrator) DebitFees(addr ethcommon.Address, manifestID ManifestI
 	orch.node.Balances.Debit(addr, manifestID, priceRat.Mul(priceRat, big.NewRat(pixels, 1)))
 }
 
-func (orch *orchestrator) isActive() (bool, error) {
+func (orch *orchestrator) isActive(addr ethcommon.Address) (bool, error) {
```

Comment:
`isActive()` still relies on the state of `orch` (calls `orch.rm.LastInitializedRound()` and `orch.node.Database.SelectOrchs()`) so alternatives to using this method are:

- `isActive(Orchestrator, Address)`
- `isActive(RoundsManager, Database, Address)`

The latter introduces additional parameters to the function signature and exposes the caller to DBFilter construction so I don't think that is better.

The only difference between the former and the current implementation is whether Orchestrator is a method receiver or function parameter. I don't see much utility in making this change so I'd prefer to just keep the method.

Since the address passed into `isActive()` is always accessible via the recipient field in a ticket I'd prefer not to save the "master" address to avoid saving additional state.